README.md

```diff
@@ -37,185 +41,198 @@ Simple parsing for node.js.
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-What is nearley?
-----------------
-nearley uses the Earley parsing algorithm augmented with Joop Leo's
-optimizations to parse complex data structures easily. nearley is über-fast and
-really powerful. It can parse literally anything you throw at it.
-
-nearley is used by [artificial
-intelligence](https://github.com/ChalmersGU-AI-course/shrdlite-course-project)
-and [computational
-linguistics](https://wiki.eecs.yorku.ca/course_archive/2014-15/W/6339/useful_handouts)
-classes at universities, as well as [file format
-parsers](https://github.com/raymond-h/node-dmi), [markup
-languages](https://github.com/bobbybee/uPresent) and [complete programming
-languages](https://github.com/bobbybee/carbon). It's an npm [staff
-pick](https://www.npmjs.com/package/npm-collection-staff-picks).
-
-Why do I care?
---------------
-
-nearley can parse what other JS parse engines cannot, because it uses a
-different algorithm. The Earley algorithm is *general*, which means it can
-handle *any* grammar you can define in BNF. In fact, the nearley syntax is
-written in *itself* (this is called bootstrapping).
+## Introduction
+
+nearley compiles grammar definitions from a simple syntax resembling [BNF](https://en.wikipedia.org/wiki/Backus–Naur_form) to a JS representation.
+You pass that representation to the nearley's tiny runtime, feed it data, and get the results.
+
+nearley uses the Earley parsing algorithm with Joop Leo's optimizations to parse complex data structures easily.
+Thanks to this algorithm, nearley can parse what other JavaScript parsers cannot.
+It can handle *any* grammar you can define in BNF.
+In fact, the nearley syntax is written in *itself* (this is called bootstrapping).
 
 PEGjs and Jison are recursive-descent based, and so they will choke on a lot of
-grammars, in particular [left recursive
-ones](http://en.wikipedia.org/wiki/Left_recursion).
+grammars, in particular [left recursive ones](http://en.wikipedia.org/wiki/Left_recursion).
 
 nearley also has capabilities to catch errors gracefully, and detect ambiguous
 grammars (grammars that can be parsed in multiple ways).
 
-Installation and Usage
-----------------------
+nearley is used by [artificial intelligence](https://github.com/ChalmersGU-AI-course/shrdlite-course-project)
+and [computational linguistics](https://wiki.eecs.yorku.ca/course_archive/2014-15/W/6339/useful_handouts)
+classes at universities, as well as [file format parsers](https://github.com/raymond-h/node-dmi),
+[markup languages](https://github.com/bobbybee/uPresent) and
+[complete programming languages](https://github.com/bobbybee/carbon).
+It's an npm [staff pick](https://www.npmjs.com/package/npm-collection-staff-picks).
```

Comment:
In particular, I think the markup language should be Idyll and the complete language should be https://github.com/sizigi/lp5562 or something.